lib/datasource/common.ts

```diff
@@ -9,7 +11,7 @@ export interface DigestConfig extends Config {
   registryUrl?: string;
 }
 
-interface ReleasesConfigBase {
+interface ReleasesConfigBase extends RenovateConfig {
```

Comment:
Isn't it enough to add `trustLevel?: string` to this interface?

Reply:
I'll do it if you want to, but my change reflects what the code actually does - the config is indeed passed all the way down to each datasource.

Reply:
Scratch that, `getPkgReleases` is called from the docker datasource also without passing down the global config, unlike most other callsites 😞